app_dart/lib/src/request_handlers/scheduler/request.dart

```diff
@@ -0,0 +1,63 @@
+// Copyright 2019 The Flutter Authors. All rights reserved.
```

Comment:
Should we call this file scheduler_request_subscription? request is too generic and it does not say much about the real purpose.